src/diffusers/pipelines/blip_diffusion/modeling_blip2.py

```diff
@@ -0,0 +1,653 @@
+# coding=utf-8
```

Comment:
Would you be able to contribute the get_image `features` and `get_text_features` methods to the Transformers library, such that this file can be omitted?

Reply:
For now, I think it's okay for the PR as it doesn't get blocked.